setup.py

```diff
@@ -203,6 +203,7 @@
             'inspire_cache = inspirehep.modules.cache.ext:INSPIRECache',
             'inspire_search = inspirehep.modules.search:INSPIRESearch',
             'inspire_workflows = inspirehep.modules.workflows:INSPIREWorkflows',
+            'invenio_collections = invenio_collections:InvenioCollections',
```

Comment:
Yes i understand this but if comment out this line travis will not complain with an error. Actually, the error will come up when you will try to make the PUT request. I can update the PR without this line but you must peek the commit and try making the request. Is it ok?